dashboard/client/src/api.ts

```diff
@@ -294,19 +294,18 @@ export const getErrors = (nodeIp: string, pid: number | null) =>
     pid: pid ?? "",
   });
 
-export type LogsResponse = {
-  logs: LogsByPid;
-};
-
-export type LogsByPid = {
-  [pid: string]: string[];
-};
-
-export const getLogs = (nodeIp: string, pid: number | null) =>
-  get<LogsResponse>("/node_logs", {
-    ip: nodeIp,
-    pid: pid ?? "",
+export const getLogs = async (nodeIp: string, pid: number) => {
+  const result = await getV2<string>("/api/v0/logs/file", {
+    params: {
+      node_ip: nodeIp,
+      pid: pid,
+      lines: 15000,
+    },
   });
+  // Substring to get rid of initial "1" or "0" that represents successful stream.
+  // TODO(sang): should we get rid of that?
```

Comment:
TODO(sang)?

Reply:
This is just to make sure we have this discussion. We can get rid of this if we make a decision that we want to continue sending the 1 or 0.